### PR TITLE
[IOTDB-859] list all files generated or used by IoTDB

### PR DESCRIPTION
**Pull request by mychaow**

### CODE REVIEW

File: docs/UserGuide/Server/ServerFileList.md [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+## Configuration Files
+> under conf directory
+1. iotdb-engine.properties
+2. logback.xml
+3. iotdb-env.sh
+4. jmx.access
+5. jmx.password
+6. iotdb-sync-client.properties
+    + only sync tool use it
+
```
Comment (HTHou) [inline docs/UserGuide/Server/ServerFileList.md:37]:
```suggestion

> under directory basedir/system/schema
1. system.properties

```

File: docs/UserGuide/System Tools/NodeTool.md [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+    under the License.
+
+-->
+
+# Introduction
+IoTDB cluster version provides nodetool shell tool for users to monitor the working status of the specified cluster. 
+Users can obtain the status of the cluster by running a variety of instructions.
+
+The following describes the usage and examples of each instruction, 
+where $IOTDB_CLUSTER_ Home indicates the path of the IoTDB Distributed installation directory.
```
Comment (LebronAl) [inline docs/UserGuide/System Tools/NodeTool.md:27]: It seems to be $IOTDB_CLUSTER_HOME~

File: docs/UserGuide/System Tools/NodeTool.md [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+> ring
+
+2.Output
+
+> The output is a multi line string, and each line of string is a key value pair, 
+> where the key represents the token value and the value represents the node (IP:METAPORT:DATAPORT), the format is \<key -> value>.
+
+3.Examples
+
+> Suppose that the current cluster runs on three nodes: 127.0.0.1:9003:40010, 127.0.0.1:9005:40012 and 127.0.0.1:9004:40011.
```
Comment (LebronAl) [inline docs/UserGuide/System Tools/NodeTool.md:58]: These three nodes are best aligned with the clockwise order of the nodes

File: docs/UserGuide/System Tools/NodeTool.md [hunk 4]
```diff
@@ -0,0 +1,288 @@
...
+|-et | --EndTime	It is used when querying data partition.<br>The end time is the current system time by default. <br> If the end time is less than the start time, the end time is the start time by default|-et 1576724778159 |
+
+2.Output
+
+> The output is a multi line string, and each line of string is a key value pair, where the key represents the partition,
+> and the value represents the data group in the format of \< key -> value>.
+
+3.Examples
+
+> Suppose that the current cluster runs on three nodes: 127.0.0.1:9003:40010, 127.0.0.1:9005:40012 and 127.0.0.1:9004:40011.
```
Comment (LebronAl) [inline docs/UserGuide/System Tools/NodeTool.md:121]: These three nodes are best aligned with the clockwise order of the nodes

File: docs/UserGuide/System Tools/NodeTool.md [hunk 5]
```diff
@@ -0,0 +1,288 @@
...
+```
+
+Press enter to execute the command. 
+
+The output of the example instruction is as follows:
+```
+DATA<root.guangzhou.d1, 1576723735188, 1576723735188>	->	[127.0.0.1:9003:40010, 127.0.0.1:9004:40011]
+```
+The above output shows that the data partition to which root.t1.d1 belongs contains two nodes,
+of which 127.0.0.1:9003:40010 is the leader node.
```
Comment (LebronAl) [inline docs/UserGuide/System Tools/NodeTool.md:182]: is this first node a leader node or header node?

File: docs/zh/UserGuide/System Tools/NodeTool.md [hunk 6]
```diff
@@ -0,0 +1,245 @@
...
+1.输入
+> 打印哈希环的指令为ring
+
+2.输出
+
+> 输出为多行字符串，每行字符串为一个键值对，其中键表示令牌值，值表示节点(IP:METAPORT:DATAPORT)，格式为\<key -> value>。
+
+3.示例
+
+> 假设当前集群运行在127.0.0.1:9003:40010、127.0.0.1:9005:40012和127.0.0.1:9004:40011三个节点上。
```
Comment (LebronAl) [inline docs/zh/UserGuide/System Tools/NodeTool.md:48]: same as above

File: docs/zh/UserGuide/System Tools/NodeTool.md [hunk 7]
```diff
@@ -0,0 +1,245 @@
...
+```
+Windows系统：
+```
+Shell > \bin\nodetool.bat -h 127.0.0.1 -p 31999 partition -path root.guangzhou.d1 -m
+```
+回车后即可执行指令。示例指令的输出如下：
+```
+DATA<root.guangzhou.d1, 1576723735188, 1576723735188>	->	[127.0.0.1:9003:40010, 127.0.0.1:9004:40011]
+```
+上述输出表示root.t1.d1所属的数据分区包含2个节点，其中127.0.0.1:9003:40010为leader节点。
```
Comment (LebronAl) [inline docs/zh/UserGuide/System Tools/NodeTool.md:154]: same as above

File: docs/UserGuide/System Tools/NodeTool.md [hunk 8]
```diff
@@ -0,0 +1,288 @@
...
+The time series metadata of iotdb cluster version is divided into multiple data partitions according to storage groups,
+in which the storage group and data partition are many to one relationship.
+
+That is, the same storage group only exists in the same data partition,
+and a data partition contains multiple storage groups.
+
+The data is divided into multiple data partitions according to the storage group and time interval,
+and the partition granularity is <storage group, time range>.
+
+The data partition is composed of replica nodes,
```
Comment (LebronAl) [inline docs/UserGuide/System Tools/NodeTool.md:97]: I think it's best to state here that a time partition consists of a raft group, thus ensuring high availability of data.

File: docs/zh/UserGuide/System Tools/NodeTool.md [hunk 9]
```diff
@@ -0,0 +1,245 @@
...
+ 
+上述输出表示当前集群共有3个节点，按照在环中的位置顺时针输出结果。
+
+### 查询数据分区和元数据分区
+IoTDB集群版的时间序列元数据按照存储组分为多个数据分区，其中存储组和数据分区为多对一的关系，
+即同一个存储组只存在于同一个数据分区，一个数据分区包含多个存储组；
+
+数据按照存储组和时间区间分为多个数据分区，分区粒度为<Storage Group，Time Range>。
+
+数据分区由多个节点组成，节点数量为副本数，其中某一个节点担任Leader的角色。
```
Comment (LebronAl) [inline docs/zh/UserGuide/System Tools/NodeTool.md:79]: same as above

**Review by LebronAl — APPROVED** [review]
Great work!
Only minor suggestions~

File: docs/UserGuide/System Tools/NodeTool.md (hunk 2 above)
Comment (mychaow) [reply docs/UserGuide/System Tools/NodeTool.md:27]: Thank you, I will fix it.

File: docs/UserGuide/Server/ServerFileList.md (hunk 1 above)
Comment (mychaow) [reply docs/UserGuide/Server/ServerFileList.md:37]: thank you!

File: docs/UserGuide/System Tools/NodeTool.md (hunk 5 above)
Comment (mychaow) [reply docs/UserGuide/System Tools/NodeTool.md:182]: header, thank you point at that.

**Review by LebronAl — APPROVED** [review]
LGTM~
BTW, It’s better to add patch based on reviews to a new commit and push it to remote branch rather than amending patch to last commit and force-update remote branch, which will make it much easier for reviewer to see your patches.

**Comment by mychaow** [comment]
> LGTM~
> BTW, It’s better to add patch based on reviews to a new commit and push it to remote branch rather than amending patch to last commit and force-update remote branch, which will make it much easier for reviewer to see your patches.

ok， thanks

**Comment by LebronAl** [comment]
> > LGTM~
> > BTW, It’s better to add patch based on reviews to a new commit and push it to remote branch rather than amending patch to last commit and force-update remote branch, which will make it much easier for reviewer to see your patches.
> 
> ok， thanks

Of course, this may cause one more commit which seems useless, so the merge manager should use `squash and merge` to squash these commits into one commit and then merge  PR into base branch.
This seems to be the more elegant way, and we can all pay attention to it in the future :)

**Comment by mychaow** [comment]
> > > LGTM~
> > > BTW, It’s better to add patch based on reviews to a new commit and push it to remote branch rather than amending patch to last commit and force-update remote branch, which will make it much easier for reviewer to see your patches.
> > 
> > 
> > ok， thanks
> 
> Of course, this may cause one more commit which seems useless, so the merge manager should use `squash and merge` to squash these commits into one commit and then merge this PR into base branch.
> This seems to be more elegant, and we can all pay attention to it in the future :)

yes, you are right.

**Review by jt2594838 — APPROVED** [review]

**Review by HTHou — APPROVED** [review]
LGTM

File: docs/zh/UserGuide/Server/ServerFileList.md [hunk 10]
```diff
@@ -0,0 +1,153 @@
...
+> conf目录下
+1. iotdb-engine.properties
+2. logback.xml
+3. iotdb-env.sh
+4. jmx.access
+5. jmx.password
+6. iotdb-sync-client.properties
+    + 只有Sync工具会使用
+
+> under directory basedir/system/schema
```
Comment (neuyilan) [inline docs/zh/UserGuide/Server/ServerFileList.md:38]: mixed with English and Chinese ^^

**Comment by jixuan1989** [comment]
cluster_new has something wrong. There is NullPointer

**Pull request merged**